README.md

```diff
@@ -15,12 +15,13 @@ Community (college) maintained list of algorithm implementations and codes.
 
 ## Resources
 
-* [Popular Data Structures and Algorithms - Codechef](https://discuss.codechef.com/questions/48877/data-structures-and-algorithms)
 * [Intro to Algorithms - Khan Academy](https://www.khanacademy.org/computing/computer-science/algorithms)
```

Comment:
Intro to Algorithms - Khan Academy should come after Fundamentals of Algorithms - GeekforGeeks